pkg/querier/querier.go

```diff
@@ -27,6 +27,9 @@ type Config struct {
 	MaxSamples               int
 	IngesterMaxQueryLookback time.Duration
 
+	// IngesterOnlyQueriesAllowed sends queries that fall within IngesterMaxQueryLookback to only ingesters and skips LTS
+	IngesterOnlyQueriesAllowed bool
```

Comment:
Where is this configured? Are you going to add flag?

Reply:
It would just be apart of the yaml config. I guess I should add the yaml annotation for clarity. I normally would have added a flag but I think the current POR is to deprecate the usage of flags for configs.

Reply:
Given @bboreham mentioned we already have such logic in the `chunk_store.go` and he agrees on moving the check in the querier, do we still need this option or we can just keep it enabled?

If we keep the option, I would be glad if you could add the CLI flag too. At Grafana we're not ready to switch to config files, while I would like to use this flag too.

Reply:
I've moved the previous config, and added a flag option.